mm2src/coins/nft/nft_errors.rs

```diff
@@ -366,3 +391,37 @@ impl From<CoinFindError> for TransferConfirmationsError {
         }
     }
 }
+
+/// Enumerates errors that can occur while clearing NFT data from the database.
+///
+/// Variants:
+/// - `DbError`: Represents errors related to database operations.
+/// - `Internal`: Indicates internal errors not directly associated with database operations.
+/// - `InvalidRequest`: Used for various types of invalid requests, such as missing or contradictory parameters.
```

Comment:
This isn't a good approach for documenting rust code. You should add the field-related docs on the fields directly.

Reply:
I see, thanks for the note! I changed as many doc comments as I've noticed. If I see more, I will fix them.